bin/test-workflow/set_env_vars.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# Set the default values of environment variables used by the scripts
+export PULL_DOCKER_REGISTRY_URL=${PULL_DOCKER_REGISTRY_URL:-${DOCKER_REGISTRY_URL}}
```

Comment:
At some point, we're going to be using this script in different scenarios/topologies (e.g. running in Google Cloud, or running in OpenShift, etc.). In most cases, this will probably be done by invoking the `start` script via Summon, with custom Summon YAML config for each scenario to set the environment variables that are used here.

Just a thought... feel free to skip this idea for now, but another mechanism that might come in handy for developers is to support a `customize.env` file similar to this one for the Conjur OSS KinD example:
    https://github.com/cyberark/conjur-oss-helm-chart/blob/master/examples/kubernetes-in-docker/customize.env
I've found this handy a few times when I just wanted to make small tweaks to how the example works.

If this sounds useful, this can be another followup issue, since it's low priority.